src/Apps/Consign/Routes/SubmissionFlow/UploadPhotos/UploadPhotos.tsx

```diff
@@ -254,6 +255,12 @@ export const UploadPhotos: React.FC<UploadPhotosProps> = ({
             refetchPhotos()
           }
 
+          const lastErrorMessage = findLast(
+            values.photos,
+            photo => photo.errorMessage
+            // @ts-ignore
```

Comment:
**question:** what are we ignoring here? 👀

Reply:
We ignoring `findLast`'s type issues... [The docs](https://lodash.com/docs/4.17.15#findLast) say this method should return the element or `undefined`.

<img width="487" alt="Screenshot 2022-10-13 at 16 11 58" src="https://user-images.githubusercontent.com/4691889/195620692-873308b4-00a3-4f72-a15e-227120ba9b89.png">